spec/features/hipstergram_features_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.feature "hipstergrams" do
+
+  before { sign_in(sign_up) }
+  
+  context "When no hipstergrams have been added" do
+
+    scenario "It should display a prompt to upload an hipstergram" do
+      visit "/hipstergrams"
+      expect(page).to have_content "You got here before it was cool; no hipstergrams have been added"
```

Comment:
Metrics/LineLength: Line is too long. [101/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)